e2e/multi-svc-app/multi_svc_app_test.go

```diff
@@ -243,16 +246,23 @@ var _ = Describe("Multiple Service App", func() {
 				// Call each environment's endpoint and ensure it returns a 200
 				route := svc.Routes[0]
 				Expect(route.Environment).To(Equal("test"))
+
+				// route.URL is of the form `example-alb.elb.us-west-2.amazonaws.com or example-nlb.elb.us-west-2.amazonaws.com, so we split to retrieve just one valid url`
+				routeURLs := strings.Split(route.URL, " ")
+				Expect(len(svc.Routes)).To(BeNumerically(">", 1))
```

Comment:
Will this really be `true`? I might have missed something - it seems like we are only deploying to `test` environment - so there will be only one route in `svc.Routes`. 

Or maybe you meant `Expect(len(routeURLs)).To(BeNumerically(">", 1))`?

Reply:
OOPS good catch thank you, I did mean `Expect(len(routeURLs)).To(BeNumerically(">", 1)`